docs/detections/alerts-run-osquery.asciidoc

```diff
@@ -35,8 +36,18 @@ image::images/setup-query.png[width=80%][height=80%][Shows how to set up the que
 TIP: To save the query for future use, click *Save for later* and define the ID,
 description, and other {kibana-ref}/osquery.html#osquery-manage-query[details].
 
-. Review the results in the table. You can also navigate to *Discover* to dive deeper into the response,
-or use the drag-and-drop *Lens* editor to create visualizations.
+. Review the results in the table. You can also:
+** Navigate to *Discover* to dive deeper into the response.
+** Use the drag-and-drop *Lens* editor to create visualizations.
+** Click the *Timeline* button (image:images/timeline-button-osquery.png[Click markdown icon,20,20]) to investigate a single query result in Timeline or *Add to timeline investigation* to investigate all results.
++
+NOTE: The option to explore Osquery results in Timeline is only available from {elastic-sec}.
++
+
++
+TIP: An `action_ID` is generated when you run an Osquery query. The `action_ID` field and value pair is passed to the Timeline's KQL filter when you select the option to open all results in Timeline.
```

Comment:
This is a slight suggestion - feel free to omit. Also assuming "action_ID" and "value pair" are two different things, hence my suggestion to change to plural. 

```suggestion
TIP: An `action_ID` is generated when you run an Osquery query. The `action_ID` field and value pair are transferred to the Timeline's KQL filter when you select the option to open all results in Timeline.
```